const/menu.ts

```diff
@@ -30,6 +30,10 @@ export const FOOTER_LINK_GROUPS: FooterLinkGroup[] = [
       { label: 'CoW Protocol', url: '/#about' },
       { label: 'CoW Swap', url: 'https://swap.cow.fi/#/about', type: 'external' },
       { label: 'CoW Swap FAQ', url: 'https://swap.cow.fi/#/faq', type: 'external' },
+      { label: 'CoW ToC', url: 'https://swap.cow.fi/#/terms-and-conditions', type: 'external' },
```

Comment:
Following Elena's comment what we probably want here is

```suggestion
      { label: 'CoW T&C', url: 'https://swap.cow.fi/#/terms-and-conditions', type: 'external' },
```

Reply:
Or use the longer version (`Terms and Conditions`) to reduce potential ambiguity

Reply:
Good point, went with full `Terms and Conditions`